src/app/shared/comcol-forms/delete-comcol-page/delete-comcol-page.component.ts

```diff
@@ -65,4 +74,23 @@ export class DeleteComColPageComponent<TDomain extends DSpaceObject> implements
   onCancel(dso: TDomain) {
     this.router.navigate([this.frontendURL + '/' + dso.uuid + '/edit']);
   }
+
+  private refreshCache(dso: TDomain) {
+    const parentCommunityUrl = this.parentCommunityUrl(dso as any);
+    if (!hasValue(parentCommunityUrl)) {
+      return;
+    }
+    this.dsoDataService.findByHref(parentCommunityUrl).pipe(
+      getSucceededOrNoContentResponse(),
+      take(1),
+    ).subscribe((rd: RemoteData<TDomain>) => {
+      const href = rd.hasSucceeded && !isEmpty(rd.payload.id) ? rd.payload.id : 'communities/search/top';
```

Comment:
It would be better to not hard-code the path to the top endpoint. But rather use the HALEndpoinService to discover it using HAL links. 

If you replace `'communities/search/top'` with `this.halService.getEndpoint('communities/search/top')` it will still work if the rest api changes its url, because that way the parts of the path ('communities', 'search' and 'top') will be treated as subsequent HAL link names to follow from the root endpoint, rather than a URL.

Reply:
Hard coded link has been replaced with HALEndpointService usage.